preprocess_segment.sh

```diff
@@ -99,33 +59,51 @@ sct_check_dependencies -short
 
 # Go to folder where data will be copied and processed
 cd $PATH_DATA_PROCESSED
+
 # Copy source images
-rsync -avzh $PATH_DATA/$SUBJECT .
+rsync -avzh $PATH_DATA/$SUBJECT/$DATA_TYPE/* .
```

Comment:
all subject files should not be in a flat directory, but instead each subject in its own directory-- to avoid file conflicts-- please do it as we do in: https://spine-generic.readthedocs.io/analysis-pipeline.html